gui/builtinPreferenceViews/pyfaHTMLExportPreferences.py

```diff
@@ -39,24 +41,24 @@ def populatePanel(self, panel):
         mainSizer.Add(self.stDesc, 0, wx.ALL, 5)
 
         self.PathLinkCtrl = wx.lib.agw.hyperlink.HyperLinkCtrl(panel, wx.ID_ANY, self.HTMLExportSettings.getPath(),
-                                             wx.DefaultPosition, wx.DefaultSize,
-                                             URL='file:///{}'.format(self.HTMLExportSettings.getPath()),)
+                                                               wx.DefaultPosition, wx.DefaultSize,
+                                                               URL='file:///{}'.format(self.HTMLExportSettings.getPath()), )
         mainSizer.Add(self.PathLinkCtrl, 0, wx.ALL | wx.EXPAND, 5)
 
-        self.fileSelectDialog = wx.FileDialog(None, "Save Fitting As...",
-                                              wildcard="EVE IGB HTML fitting file (*.html)|*.html", style=wx.FD_SAVE)
+        self.fileSelectDialog = wx.FileDialog(None, _t("Save Fitting As..."),
+                                              wildcard=_t("EVE IGB HTML fitting file (*.html)|*.html"), style=wx.FD_SAVE)
```

Comment:
Same thing as the first comment. Will probably just want to find all `wildcard=` and update them 👍